# HG changeset patch
# Parent c73db4e30bec15f0c81a6c4111026ed5a0a16d66
# User Ivan Krasin <krasin@chromium.org>
llvm/lib/Support/MemoryBuffer.cpp: revert SRPC file access.    
Review URL: http://codereview.chromium.org/7483010/
BUG=http://code.google.com/p/nativeclient/issues/detail?id=1810

diff -r c73db4e30bec -r e56fa07c58c0 tools/llc/llc.cpp
--- a/tools/llc/llc.cpp	Fri Jul 15 14:35:50 2011 -0700
+++ b/tools/llc/llc.cpp	Fri Jul 15 15:12:12 2011 -0700
@@ -245,7 +245,7 @@
   FOS.flush();
   ROS.flush();
 
-#if defined(__native_client__)
+#if defined(__native_client__) && defined(NACL_SRPC)
   NaClOutputStringToFile(filename, s);
 #else
   std::string error;
